<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COperations</class>
 <widget class="QWidget" name="COperations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Dal:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="deDal"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Al:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="deAl"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tvMain">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="toolTip">
        <string>modifica registrazione selezionata</string>
       </property>
       <property name="text">
        <string>Modifica</string>
       </property>
       <property name="icon">
        <iconset resource="graphics.qrc">
         <normaloff>:/Resources/Pencil.PNG</normaloff>:/Resources/Pencil.PNG</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="graphics.qrc"/>
 </resources>
 <connections/>
</ui>
